src/views/v2/components/breakpoints.php

```diff
@@ -20,32 +20,49 @@
 }
 ?>
 <script class="tribe-events-breakpoints">
-	(function(){
-		if ( 'undefined' === typeof window.tribe ) {
-			return;
-		}
+	(function () {
```

Comment:
we put spaces after and before parentheses for JS:
```suggestion
	( function () {
```